### PR TITLE
Get the string ip from peer

### PR DESCRIPTION
**Pull request by ilyn1337**
If you're using peer->address to get IP, it'll be like this: 169715255
If you're using this way, IP will be like this: 172.217.18.110

Usage: ((PlayerInfo*)(peer->data))->charIP

### CODE REVIEW

**Comment by iRestartz** [comment]
ty

**Comment by Spotify1337** [comment]
Testing for leet IP

**Comment by playingo** [comment]
nice one, noob :+1:

**Comment by playingo** [comment]
somehow crashes for me, did u try it out before releasing it or did i just do something wrong?

**Comment by ilyn1337** [comment]
someone* crashes you, i was using this code...

**Comment by arshizx** [comment]
> someone* crashes you, i was using this code...

i tried but didn't work idk

**Comment by playingo** [comment]
ilyn1337 someone? i tested it out in localhost

**Comment by arshizx** [comment]
> ilyn1337 someone? i tested it out in localhost

work?

**Comment by ilyn1337** [comment]
what is the error with code

**Comment by ilyn1337** [comment]
@arshizx why ur putting &????

**Comment by arshizx** [comment]
still blank like ""

File: enet server test/enet server test.cpp [hunk 1]
```diff
@@ -2147,6 +2148,9 @@ int _tmain(int argc, _TCHAR* argv[])
 			}
 
 			event.peer->data = new PlayerInfo;
+			/* Get the string ip from peer */
+			char clientConnection[32];
```
Comment (GrowtopiaNoobs) [inline enet server test/enet server test.cpp]: 16 bytes should be enough.
```suggestion
			char clientConnection[16];
```

File: enet server test/enet server test.cpp [hunk 2]
```diff
@@ -2147,6 +2148,9 @@ int _tmain(int argc, _TCHAR* argv[])
...
+			((PlayerInfo*)(peer->data))->charIP = enet_address_get_host_ip(&peer->address, clientConnection, 32);
```
Comment (GrowtopiaNoobs) [inline enet server test/enet server test.cpp]: `enet_address_get_host_ip` doesn't return text (it returns if function failed or not)
```suggestion
			enet_address_get_host_ip(&peer->address, clientConnection, 16);
			((PlayerInfo*)(peer->data))->charIP = clientConnection;
```

**Comment by GrowtopiaNoobs** [comment]
So, can someone confirm this works?

**Comment by Yoayao** [comment]
work

**Comment by flossboss** [comment]
@ilyn1337 good job my dude 👍 super useful

**Pull request merged**